src/ctia/stores/es/init.clj

```diff
@@ -14,33 +14,45 @@
    :indexname s/Str
    :shards s/Num
    :replicas s/Num
+   (s/optional-key :write-suffix) s/Str
    s/Keyword s/Any})
 
 (def store-mappings
   (apply merge {}
          (map (fn [[_ {:keys [entity es-mapping]}]]
-                {entity es-mapping}) entities)))
+                {entity es-mapping})
+              entities)))
 
 (s/defn init-store-conn :- ESConnState
   "initiate an ES store connection, returning a map containing a
    connection manager and dedicated store index properties"
-  [{:keys [entity indexname shards replicas mappings]
+  [{:keys [entity indexname shards replicas mappings aliased]
+    :or {aliased false}
     :as props} :- StoreProperties]
-  (let [settings {:number_of_shards shards
+  (let [write-alias (str indexname
+                         (when aliased "-write"))
+        settings {:number_of_shards shards
                   :number_of_replicas replicas}]
     {:index indexname
-     :props props
+     :props (assoc props :write-alias write-alias)
```

Comment:
The `write-alias` property should only be used when the an alias is effectively used. Otherwise it can be confusing